docs/manual/install/index.rst

```diff
@@ -2,6 +2,15 @@
 Installing Qtile
 ================
 
+TL;DR:
+
+.. code-block:: sh
```

Comment:
```suggestion
.. code-block:: sh

```

Reply:
Um... The TL;DR is empty. I think you need a space here.

Reply:
Ah oops, thanks!